kv/db.go

```diff
@@ -123,7 +111,8 @@ func (s *DBServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	method = strings.TrimPrefix(method, DBPrefix)
-	if !proto.IsPublic(method) {
```

Comment:
What happened to IsPublic? Is it now implicit in the fact that createArgsAndReply only knows about public methods? This should be documented loudly (and tested) since it would be bad if someone re-added the internal methods to createArgsAndReply. If this is the only remaining use of createArgsAndReply we should probably move it from `kv` to `server`.